cordapp-example/contracts-kotlin/src/main/kotlin/com/example/state/CashState.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.state
+
+import com.example.contract.CashContract
+import com.example.schema.CashSchemaV1
+import net.corda.core.contracts.Amount
+import net.corda.core.contracts.BelongsToContract
+import net.corda.core.contracts.CommandAndState
+import net.corda.core.contracts.OwnableState
+import net.corda.core.identity.AbstractParty
+import net.corda.core.schemas.MappedSchema
+import net.corda.core.schemas.PersistentState
+import net.corda.core.schemas.QueryableState
+import java.util.*
+
```

Comment:
@xavierlepretre I left a lot of comments on the first revision of `CashState` talking about `issuer`, `holder`, and `quantity` fields; I can't find that commit.
Was there a discussion somewhere where you advised the below approach? (i.e. no `issuer`, and using the name `balance` which is not correct, because this is not the balance of the `owner`), etc...Can you post the link to the discussion that lead to the below code?

Reply:
I found my comments: https://github.com/DZariusz/cordapp-template-kotlin/pull/1

@xavierlepretre did you advise against those remarks, or they were simply ignored?

Reply:
Sorry, just trying to be up to date; since I don't always follow the conversation between the 2 of you 🙂

Reply:
I don't think I advised against `issuer`. Instead I advise for it in this repo.

Reply:
Alright cool 🙂 I was just a bit confused and thought maybe you guys are taking a simpler path in implementing Cash.